src/corecel/CMakeLists.txt

```diff
@@ -236,6 +235,7 @@ list(APPEND PRIVATE_DEPS corecel_openmp_obj)
 
 # All these classes depend on MPI if available
 celeritas_add_object_library(corecel_mpi_obj
+  Assert.cc
```

Comment:
@stognini This should fix the MPI issue caused by the previous mR)

Reply:
It does indeed. Thanks!